lib/presentation/widgets/custom_icon_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:trip_tally/presentation/theme/app_colors.dart';
+
+import '../theme/app_dimensions.dart';
+
+class CustomIconButton extends StatelessWidget {
```

Comment:
As there are SVG pictures in the button this name can be misleading. Maybe `CircleElevatedButton` or `CircleSvgButton`